packages/x-components/src/x-modules/query-suggestions/components/query-suggestions.vue

```diff
@@ -67,6 +68,14 @@
      */
     @Prop()
     protected animation!: Vue;
+
+    /**
+     * Number of query suggestions to be rendered.
+     *
+     * @public
+     */
+    @Prop({ default: 5 })
```

Comment:
Like I said above, I wouldn't set a max number of items here. This will make this component not able to render all the available suggestions. Or we can accept this and say, hey if you want to render all the available suggestions, just set a big number.